3-mcstas/SANS_from_function.ipynb

```diff
@@ -164,7 +159,22 @@
    },
    "outputs": [],
    "source": [
-    "# quiz C"
+    "quiz.question_1()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "disciplinary-cause",
+   "metadata": {
+    "tags": [
+     "solution",
+     "hide-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "quiz.question_1(\"C\")"
```

Comment:
Since your quiz provides feedback and the correct answers, maybe it is not necessary to have the solutions inside the notebook? Even if they get stripped, maybe it's not needed to have the answers in the online docs?

Reply:
For multiple choice, I would agree. But for other questions, like the one you show a screenshot below of, the feedback doesn't tell you the correct answer. So I'd leave the solutions in.

Reply:
I think I only removed solutions to multiple choice questions?

Reply:
Good call, no need for that in the book.